js/sdk/src/components/ReleaseCreate.js

```diff
@@ -134,15 +136,24 @@ const ReleaseCreate = ({ canAddContent, hubPubkey }) => {
     [bundlrBalance, solPrice]
   )
   useEffect(() => {
-    refreshBundlr()
-    getUserBalances()
+    const handleInitialBalanceCheck = async () => {
+      setSolBalanceFetched(false)
+      await refreshBundlr()
+      await getUserBalances()
+      setLocalBalanceFetched(true)
```

Comment:
do we want to setSolBalanceFetched(true) here? or does it not matter

Reply:
`setSolBalanceFetched(true)` is handled in the `getUserBalances` function, doesnt have to be explicitly redelcared to true here